### PR TITLE
ci: jenkins: fix path to manage_ctr_mgr.sh

### PR DESCRIPTION
**Pull request by jcvenegas**
Fix path to manage_ctr_mgr.sh

Fixes: #907

Signed-off-by: Jose Carlos Venegas Munoz <jose.carlos.venegas.munoz@intel.com>

### CODE REVIEW

File: .ci/jenkins_job_build.sh [hunk 1]
```diff
@@ -82,7 +82,7 @@ fi
 # Make sure runc is default runtime.
 # This is needed in case a new image creation.
 # See https://github.com/clearcontainers/osbuilder/issues/8
-"${GOPATH}/${tests_repo}/cmd/container-manager/manage_ctr_mgr.sh" docker configure -r runc  -f || true
+"${GOPATH}/src/${tests_repo}/cmd/container-manager/manage_ctr_mgr.sh" docker configure -r runc  -f
```
Comment (grahamwhaley) [inline .ci/jenkins_job_build.sh]: If I were being picky :-), I think you have a 'double space' at the end before the `-f` :-)

**Comment by grahamwhaley** [comment]
oh, that's interesting, the metrics ci failed (which is something this is fixing...)

**Comment by jcvenegas** [comment]
@grahamwhaley  I see still fail for the same reason... 
.ci/jenkins_job_build.sh: line 85: /srv/jenkins/workspace/workspace/clear-containers-tests-16.04-PR/go/github.com/clearcontainers/tests/cmd/container-manager/manage_ctr_mgr.sh: No such file or directory

the path is not updated yet , because cloned master?

**Comment by jcvenegas** [comment]
@chavafg  what is the issue on  k8s ci `kubernetes qa-failed`?

**Comment by grahamwhaley** [comment]
@jcvenegas - heh heh, I think it is even a little more evil than that....
we start by checking out the master branch, and then run the `.ci/jenkins_job_build.sh` from that branch - and that script then switches branches.... but, it is 'too late', as we are already running that script, so we won't notice the update from the branch switch.

Thus, I'm happy the change is correct, and we understand why the metrics CI is not passing - I'm happy if we merge this, and re-visit if we are wrong and have to fix again...

**Pull request merged**